feat: review average fetching improvement (#673)
plentymarkets/plentyshop-pwa: docs/changelog/changelog_en.md

- PayPal: We now use card fields instead of hosting fields for credit card payments.
- Change the file item property color to blue in the quick checkout.
- Changed the name of the Cloudflare Turnstile Sitekey environment variable to make it compatible with the system configuration.

> - Optimize review average fetch by eliminating it and getting the data from the reviews request instead.

Reviewer: ```suggestion
- Eliminate the separate review average fetch by retrieving data directly from the reviews request.
```
